feature-pack/src/main/resources/configuration/domain/template.xml

```diff
@@ -96,20 +97,31 @@
                 <extension module="org.wildfly.extension.discovery"/>
                 <extension module="org.wildfly.extension.ee-security"/>
                 <extension module="org.wildfly.extension.elytron"/>
+                <extension module="org.wildfly.extension.microprofile.config-smallrye" />
             </excluded-extensions>
         </host-exclude>
         <host-exclude name="WildFly11.0">
             <host-release id="WildFly11.0"/>
             <excluded-extensions>
                 <extension module="org.wildfly.extension.ee-security"/>
+                <extension module="org.wildfly.extension.microprofile.config-smallrye" />
             </excluded-extensions>
         </host-exclude>
         <host-exclude name="WildFly12.0">
             <host-release id="WildFly12.0"/>
             <excluded-extensions>
                 <extension module="org.wildfly.extension.ee-security"/>
+                <extension module="org.wildfly.extension.microprofile.config-smallrye" />
             </excluded-extensions>
         </host-exclude>
+        <!--
+        <host-exclude name="WildFly13.0">
```

Comment:
https://github.com/wildfly/wildfly-core/pull/3416

Reply:
I will uncomment that in a subsequent PR as there is not yet a Core release with that key